src/range_proof.rs

```diff
@@ -16,13 +16,26 @@ use serde::{de::Visitor, Deserialize, Deserializer, Serialize, Serializer};
 
 use crate::{
     errors::ProofError,
+    generators::pedersen_gens::ExtensionDegree,
     inner_product_round::InnerProductRound,
     protocols::{scalar_protocol::ScalarProtocol, transcript_protocol::TranscriptProtocol},
     range_statement::RangeStatement,
     range_witness::RangeWitness,
-    utils::generic::{bit_vector_of_scalars, nonce, read32},
+    utils::generic::{bit_vector_of_scalars, nonce, read32, read8},
+    PedersenGens,
 };
 
+/// Optionally extract masks when verifying the proofs
+#[derive(Copy, Clone, Debug, PartialEq)]
+pub enum ExtractMasks {
+    /// No masks will be recovered (e.g. as a public entity)
+    NO,
```

Comment:
I would have thought Clippy would complain about this naming. Perhaps we're missing a rule in lints

Reply:
I have added `'clippy::upper_case_acronyms',`, but to no avail, not picked up.